src/transform/split-declaraion.js

```diff
@@ -0,0 +1,28 @@
+import traverser from '../traverser';
+import multiReplaceStatement from '../utils/multi-replace-statement';
+import VariableDeclaration from '../syntax/variable-declaration';
+
+export default function(ast) {
+  traverser.replace(ast, {
```

Comment:
As we're not returning the replacement node, we should use `traverser.traverse()` instead of `traverser.replace()`.